src/views/DocumentationTopic.vue

```diff
@@ -42,39 +56,50 @@
                       @close="handleToggleSidenav(breakpoint)"
                     />
                   </transition>
-                </template>
-              </NavigatorDataProvider>
+              </template>
+            </NavigatorDataProvider>
+          </div>
+        </template>
+        <Nav
+          v-if="!isTargetIDE"
+          :title="topicProps.title"
+          :diffAvailability="topicProps.diffAvailability"
+          :interfaceLanguage="topicProps.interfaceLanguage"
+          :objcPath="objcPath"
+          :swiftPath="swiftPath"
+          :parentTopicIdentifiers="parentTopicIdentifiers"
+          :isSymbolDeprecated="isSymbolDeprecated"
+          :isSymbolBeta="isSymbolBeta"
+          :currentTopicTags="topicProps.tags"
+          :references="topicProps.references"
+          :isWideFormat="enableNavigator"
+          :sidenavHiddenOnLarge="sidenavHiddenOnLarge"
+          @toggle-sidenav="handleToggleSidenav"
+        >
+          <template #menu-items>
+            <div
```

Comment:
Why use a div if you want a clickable element? Just use a `<button>` ?

Reply:
Resolved 4bd2fd13e893ecb2b3dee9617a6ba7e430882f48